diff --git a/test/experimental/eventtype_autocreate_test.go b/test/experimental/eventtype_autocreate_test.go
index 52ed72876..37186dcda 100644
--- a/test/experimental/eventtype_autocreate_test.go
+++ b/test/experimental/eventtype_autocreate_test.go
@@ -59,6 +59,7 @@ func TestBrokerEventTypeAutoCreate(t *testing.T) {
 }
 
 func TestPingSourceEventTypeMatch(t *testing.T) {
+	t.Skip("Skip, while SO deployes a eventing release, which does not support EventTypes v1beta2")
 	t.Parallel()
 
 	ctx, env := global.Environment(
